completions/apt-get

```diff
@@ -34,11 +34,14 @@ _apt_get()
                     _filedir deb
                     return
                 elif [[ $cur == *=* ]]; then
-                    COMPREPLY=( $(compgen -W "$(\
-                        apt-cache --no-generate show "${cur%%=*}" 2>/dev/null |
-                        command sed -ne \
-                            's/^Version:[[:space:]]*\([^[:space:]]\)/\1/p')" \
-                        -- "${cur#*=}") )
+                    IFS='=' read -r package cur <<< "$cur"
```

Comment:
Two remaining notes on this line:
- `package` should quite likely be made `local`
- herestrings i.e. `<<<` use temporary files, would be better to find a way to implement this without them

Reply:
For example something like (untested):
```
local package=${cur%%=*}
cur=${cur#*=}
```